### ISSUE TITLE
Cannot load 'parseconfig'

### ISSUE THREAD

**Issue by noahzgordon** [dmathieu/heroku-buildpack-submodules#3]
Today, our automatic deploys using this buildpack began failing.

```
-----> GIT Submodules app detected
---> Installing parseconfig gem
ERROR:  While executing gem ... (Errno::EROFS)
    Read-only file system - /var/lib/gems
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require': cannot load such file -- parseconfig (LoadError)
	from /usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
	from /app/tmp/buildpacks/0207a557e9ab3c6d82dc57bf3286747e0fdfede15ca080df30b70ba4a5d01ed64abea8b23efbf70f0c9d31e290dcf639f2e4accab00a17982b71bea4c21a96fe/bin/install_git_submodules.rb:1:in `<main>'
 !     Push rejected, failed to compile GIT Submodules app.
 !     Push failed
```

It looks like /var/lib/gems is now readonly, so we can't install the `parseconfig` gem. Is anyone else experiencing this?

**Comment by dmathieu** [comment]
Thank you. Fixed in #4.

**Comment by noahzgordon** [comment]
@dmathieu thanks for the quick fix!
